src/utils/defuse.ts

```diff
@@ -12,3 +12,8 @@ export type DefuseUserId = string & { __brand: "DefuseAccountId" }
 export function userAddressToDefuseUserId(userAddress: string): DefuseUserId {
   return userAddress.toLowerCase() as DefuseUserId
 }
+
+// Branding a string type with a unique symbol to prevent accidental misuse.
+// Used in settings.ts to ensure RPC URLs are properly typed and validated.
+// See settings.ts for example usage with different blockchain networks.
+export type RpcUrl = string & { __brand: "RpcUrl" }
```

Comment:
I don't think we need branded string for URL. Branded types should be used when it's critical to use nominal types. But in our case it creates more friction than needed, say we allow users to override RPC URLs, then how would users get this type? They will just typecast (`as RpcUrl`, btw you do the same), which does not help with type safety. I suggest getting rid of it and use just `string`.

Also it's not a `defuse` specific thing, so if you insist that we need this branded string, then it should be moved somewhere else.

Reply:
Agree, cool points!